src/test/regress/expected/distributed_transactions.out

```diff
@@ -0,0 +1,1233 @@
+DROP SCHEMA IF EXISTS distributed_transaction_function CASCADE;
+NOTICE:  schema "distributed_transaction_function" does not exist, skipping
+CREATE SCHEMA distributed_transaction_function;
+SET search_path TO 'distributed_transaction_function';
+SET citus.shard_replication_factor = 1;
+SET citus.shard_count = 32;
+SET citus.next_shard_id TO 900000;
+CREATE TABLE test_txn_dist(intcol int PRIMARY KEY);
+SELECT create_distributed_table('test_txn_dist', 'intcol', colocate_with := 'none');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE FUNCTION insert_data(a integer, b integer)
+RETURNS void LANGUAGE plpgsql AS $fn$
+BEGIN
+	INSERT INTO distributed_transaction_function.test_txn_dist VALUES (a);
+	INSERT INTO distributed_transaction_function.test_txn_dist VALUES (b);
+END;
+$fn$;
+SELECT create_distributed_function(
+  'insert_data(int, int)', 'a',
+  colocate_with := 'test_txn_dist',
+  force_pushdown := true
+);
+ create_distributed_function
+---------------------------------------------------------------------
+
+(1 row)
+
+-- make sure that metadata is synced, it may take few seconds
+CREATE OR REPLACE FUNCTION wait_until_metadata_sync(timeout INTEGER DEFAULT 15000)
```

Comment:
can just put this test file in a turn_mx_on part of the test schedule? then the metadata is always synced

Reply:
Done